projects/BFDR/FetalDeathRecord.cs

```diff
@@ -68,6 +68,104 @@ protected override void InitializeCompositionAndSubject()
             Composition.Title = "Fetal Death Report";
         }
 
+        /// <summary>Fetus Legal Name - Given. Middle name should be the last entry.</summary>
+        /// <value>the fetus' name (first, etc., middle)</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>string[] names = { "Example", "Something", "Middle" };</para>
+        /// <para>ExampleFetalDeathRecord.FetusGivenNames = names;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Fetus Given Name(s): {string.Join(", ", ExampleFetalDeath.FetusGivenNames)}");</para>
+        /// </example>
+        [Property("FetusGivenNames", Property.Types.StringArr, "Fetus Name", "Fetus' First Name.", true, BFDR.IGURL.PatientDecedentFetus, true, 0)]
+        [FHIRPath("Bundle.entry.resource.where($this is Patient)", "name")]
+        public string[] FetusGivenNames
+        {
+            get
+            {
+                return Subject?.Name?.Find(name => name.Use == HumanName.NameUse.Official)?.Given?.ToArray() ?? new string[0];
+            }
+            set
+            {
+                updateGivenHumanName(value, Subject.Name);
+            }
+        }
+
+        /// <summary>Fetus' Legal Name - Last.</summary>
+        /// <value>the fetus' last name</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>string lastName = "Quinn";</para>
+        /// <para>ExampleFetalDeathRecord.FetusFamilyName = lastName;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Fetus Family Name(s): {string.Join(", ", ExampleFetalDeathRecord.FetusFamilyName)}");</para>
+        /// </example>
+        [Property("FetusFamilyName", Property.Types.String, "Fetus Name", "Fetus' Last Name.", true, BFDR.IGURL.PatientDecedentFetus, true, 0)]
+        [FHIRPath("Bundle.entry.resource.where($this is Patient)", "name")]
+        public string FetusFamilyName
+        {
+            get
+            {
+                return Subject?.Name?.Find(name => name.Use == HumanName.NameUse.Official)?.Family;
+            }
+            set
+            {
+                HumanName name = Subject.Name.SingleOrDefault(n => n.Use == HumanName.NameUse.Official);
```

Comment:
Could this code could be abstracted to a helper method Like `GetFamilyName()` which takes a patient as an argument? Then it could share common code with ChildFamilyName and MotherFamilyName, updating those accordingly.